<launch>
    <arg name="num_cars" default="2" />
    <node name="launchcars" pkg="mushr_sim" type="multi_car.py">
        <param name="num_cars" type="int" value="$(arg num_cars)"/>
    </node>
</launch>
